doc/user/content/integrations/_index.md

```diff
@@ -33,22 +33,22 @@ For listed tools that are not yet production-ready, you can register your intere
 
 ### Kafka
 
-Kafka is supported as a [**source**](/overview/key-concepts/#sources), with features like **upserts** and **Debezium** CDC.
+Kafka is supported as a [**source**](/overview/key-concepts/#sources) and as a [**sink**](/overview/key-concepts/#sinks), with features like **upserts**, **Debezium** CDC, and **exactly-once** processing.
```

Comment:
Trying to think how to phrase this in a way that doesn't highlight EO as a feature anymore, but as the default. 🤔

Reply:
Rephrased!